changelogs/unreleased/82115.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "SupplierArrivalLineDetailScreen: possibility to add tracking number without verification",
```

Comment:
Supplier arrival: add possibility to register a tracking number when the line is incomplete.